data/db-schema/src/main/resources/net/corda/db/schema/vnode-vault/migration/vnode-vault-creation-v1.0.xml

```diff
@@ -88,6 +88,9 @@
             <column name="member_name" type="NVARCHAR(255)">
                 <constraints nullable="false"/>
             </column>
+            <column name="is_pending" type="BOOLEAN">
+                <constraints nullable="false"/>
+            </column>
             <column name="status" type="NVARCHAR(64)">
```

Comment:
Do we need both `status` and `is_pending`? Or can we just add `status` to the primary key?

Reply:
I think is_pending was added to have less entries, because when we will have suspended as well then it won't overwrite the active member info, because of the keying strategy, so effectively we will have unnecessary data in the db.

Reply:
When we have suspended shouldn't we remove the active version? So we can only have a pending member info and either and active or a suspended but not all three? Or at least a soft delete (i.e. the is_deleted column) so that the reconciler (when added) will remove the stale entries from the DB even when keying with status.

Reply:
I think here: https://r3-cev.atlassian.net/wiki/spaces/CB/pages/4345921553/MGM+keying+strategy+for+membership+list
We decided on approach number 3 without any deletion/clean-up.
We can review the approach if we want.